SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -456,6 +442,40 @@ void USpatialActorChannel::GetLatestAuthorityChangeFromHierarchy(const AActor* H
 	}
 }
 
+void USpatialActorChannel::MarkActorHierarchyForMigration(USpatialNetDriver* NetDriver, VirtualWorkerId NewAuthVirtualWorkerId, AActor* HierarchyActor)
+{
+	if (HierarchyActor->GetIsReplicated() && HierarchyActor->HasAuthority())
```

Comment:
What would happen if you try to migrate while waiting for something in your hierarchy to get authority?

Reply:
Or are we just ignoring such cases until someone deals with working sets?

Reply:
The case I was seeing was more an actor spawned on another worker trying to "join" its hierarchy. So it should definitely move.
This should be mentioned as a corner case of https://improbableio.atlassian.net/browse/UNR-3539